src/containers/EventListener/EventListener.js

```diff
@@ -93,14 +97,25 @@ export /* istanbul ignore next */ class EventListenerContainer extends Component
             {serviceType}
           </p>
         )}
+        {namespaceSelector?.matchNames?.length ? (
```

Comment:
Can we use `&&` instead of the conditional ?

Reply:
when dealing with a numeric condition it's safer to use the ternary otherwise if you're not careful you can end up with a rogue '0' appearing on the page